### DISCUSSION TITLE
High CLS (layout shift) when scrolling

### DISCUSSION THREAD

**Discussion by macnux** [fomantic/Fomantic-UI#1877] [General] [answered]
# Bug Report
Is there is a solution to avoid the high CLS (layout shift) when scrolling?

## Steps to reproduce
Just scroll down and the CLS value will increase.

## Testcase
https://fomantic-ui.com/modules/sticky.html

**Comment by lubber-de** [comment, marked as answer]
From my understanding a perfect CLS is about reserving exact space for content which is lazy loaded, so space does not change when the area appears. (Please correct me, if i am mistaken.)

Isn't this something the webpage designer has to take care of? Rather than the framework?
You may use [placeholder](https://fomantic-ui.com/elements/placeholder.html) for that purpose.

Other than that, i cannot see any issues when scrolling the mentioned sticky demopage (which is only there to demonstrate the sticky module anyway)

**Comment by lubber-de** [comment]
Moving to discussion, because this is a question of individual page layout and not a framework bug.
High CSL on the sticky demo page is natural, because changing content position is the general usecase of the sticky module
